build.ps1

```diff
@@ -933,28 +937,37 @@ function Build-SourceKitLSP($Arch)
 function Build-Installer()
 {
   # Currently fails due to _InternalSwiftScan paths
-  # Build-WiXProject toolchain.wixproj -Properties @{
+  # Build-WiXProject toolchain.wixproj -Arch $HostArch -Properties @{
   #   DEVTOOLS_ROOT = "$ToolchainInstallRoot\";
   #   TOOLCHAIN_ROOT = "$ToolchainInstallRoot\";
   # }
 
-  # TODO: The XCTest depends on the architecture
-  # Build-WiXProject sdk.wixproj -Properties @{
-  #   PLATFORM_ROOT = "$PlatformInstallRoot\";
-  #   SDK_ROOT = "$SDKInstallRoot\";
-  #   SWIFT_SOURCE_DIR = "$SourceCache\swift\";
-  # }
+  foreach ($Arch in $SDKArchs)
+  {
+    # WiX v3 does not support ARM64
+    if ($Arch -eq $ArchARM64)
+    {
+      continue
+    }
 
-  Build-WiXProject runtime.wixproj -Properties @{
-    SDK_ROOT = (Get-RuntimeInstallDir $ArchX64) + "\";
+    Build-WiXProject runtime.wixproj -Arch $Arch -Properties @{
+      SDK_ROOT = (Get-RuntimeInstallDir $ArchX64) + "\";
```

Comment:
That should be `$Arch` not `$ArchX64` as the runtimes are architecture dependent.

Reply:
Good call. Fixed in latest